package.json

```diff
@@ -82,18 +82,19 @@
     "npm-run-all": "^4.1.5",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
     "polished": "^3.4.4",
-    "postcss-loader": "^3.0.0",
```

Comment:
Why is postcss removed here?

Reply:
Because we don't use it at all. We can use RTLCSS without it.